automated install/basic-install.sh

```diff
@@ -86,8 +86,10 @@ if [ -x "$(command -v apt-get)" ]; then
 	PKG_INSTALL="${PKG_MANAGER} --yes --fix-missing install"
 	# grep -c will return 1 retVal on 0 matches, block this throwing the set -e with an OR TRUE
 	PKG_COUNT="${PKG_MANAGER} -s -o Debug::NoLocking=true upgrade | grep -c ^Inst || true"
+	# Debian 7 doesn't have iproute2 use iproute
+	${PKG_MANAGER} install --dry-run iproute2 > /dev/null 2>&1 && IPROUTE_PKG='iproute2' || IPROUTE_PKG='iproute'
```

Comment:
You did a pull earlier that had the PHP dependency check in  the same section, could you add that to this., please?